### PR TITLE
[OSDOCS-4102] Application credentials RN for RHOSP clouds.yaml

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.12

Issue: [OSDOCS-4102](https://issues.redhat.com/browse/OSDOCS-4102)

Link to docs preview: https://53623--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-auth

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by MaysaMacedo** [comment]
From QE side:
/cc @imatza-rh

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -446,6 +446,42 @@ In {product-title} {product-version}, by enabling C-states and OS-controlled P-s
...
+    openstack:
+        auth:
+            auth_url: https://127.0.0.1:13000
+            application_credential_id: '5dc185489adc4b0f854532e1af81ffe0'
+            application_credential_secret: 'PDCTKans2bPBbaEqBLiT_IajG8e5J_nJB4kvQHjaAy6ufhod0Zl0NkNoBzjn_bWSYzk587ieIGSlT11c4pVehA'
+        auth_type: "v3applicationcredential"
+        region_name: regionOne
+----
+
+To use application credentials with your cluster, as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster, or create the `clouds.yaml` file and rotate it into an existing cluster.
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:482]:
```suggestion
To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster, or create the `clouds.yaml` file and rotate it into an existing cluster.
```

**Review by skrthomas — COMMENTED** [review]
A couple of small nits, LGTM otherwise

**Comment by stephenfin** [comment]
Other than Sara's comments, this LGTM. Will approve once those are fixed.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -446,6 +446,42 @@ In {product-title} {product-version}, by enabling C-states and OS-controlled P-s
...
+    openstack:
+        auth:
+            auth_url: https://127.0.0.1:13000
+            application_credential_id: '5dc185489adc4b0f854532e1af81ffe0'
+            application_credential_secret: 'PDCTKans2bPBbaEqBLiT_IajG8e5J_nJB4kvQHjaAy6ufhod0Zl0NkNoBzjn_bWSYzk587ieIGSlT11c4pVehA'
+        auth_type: "v3applicationcredential"
+        region_name: regionOne
+----
+
+To use application credentials with your cluster, as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
```
Comment (maxwelldb) [inline release_notes/ocp-4-12-release-notes.adoc:482]:
```suggestion
To use application credentials with your cluster as a {rh-openstack} administrator, create the credentials. Then, use them in a `clouds.yaml` file when you install a cluster. Alternatively, you can create the `clouds.yaml` file and rotate it into an existing cluster.
```

**Comment by maxwelldb** [comment]
@stephenfin @MaysaMacedo LMK by the end of the week if you'd like to see any other changes to this.

**Comment by maxwelldb** [comment]
@imatza-rh Ready for QE.

**Review by imatza-rh — COMMENTED** [review]
/lgtm

**Comment by mandre** [comment]
@maxwelldb There's [a known issue](https://issues.redhat.com/browse/OCPBUGS-5036) where the external cloud provider doesn't pick up the updated credentials after a rotation. Should it be mentioned in the release notes? 
cc @pierreprinetti

**Comment by maxwelldb** [comment]
https://github.com/openshift/openshift-docs/pull/53623#issuecomment-1373406622

Let's track discussion for this in #54280

**Pull request merged**